test/sdk.test.ts

```diff
@@ -158,10 +158,33 @@ describe('SDK Init', () => {
 
   test('RewriteFrames to be added by default', async () => {
     NATIVE.platform = 'web';
+    init({ enabled: true }, async (capacitorOptions: CapacitorOptions) => {
+      const rewriteFramesIntegration =
+        Array.isArray(capacitorOptions.integrations) &&
+        capacitorOptions.integrations.find(
+          (integration) => integration.name === 'RewriteFrames');
 
-    init({ enabled: true }, (capacitorOptions: CapacitorOptions) => {
-      expect(capacitorOptions).toBeDefined();
-      expect(capacitorOptions.integrations).toContain(capacitorRewriteFramesIntegration);
+      // Capacitor specific frame.
+      const error = { values: [{ stacktrace: { frames: [{ filename: 'capacitor://localhost:8080/file.js' }] } }] };
+      const expectedError =
+      {
+        filename: 'app:///file.js',
+        in_app: true
+      }
+
+      expect(rewriteFramesIntegration).toBeDefined();
+
+      if (rewriteFramesIntegration == false) {
+        throw new Error('rewriteFrames should be defined, but it is false');
+      }
```

Comment:
This is safer to use, as the variable can be false from `isArray` but also undefined from the `find` call.

```suggestion
 if (!rewriteFramesIntegration) {
```